common/lib/xmodule/xmodule/js/src/video/07_video_volume_control.js

```diff
@@ -39,12 +39,24 @@ function() {
 
         template: [
             '<div class="volume">',
-                '<a href="#" role="button" aria-disabled="false" title="',
-                    gettext('Volume'), '" aria-label="',
-                    gettext('Click on this button to mute or unmute this video or press UP or DOWN buttons to increase or decrease volume level.'),
-                    '"></a>',
-                '<div role="presentation" class="volume-slider-container">',
-                    '<div class="volume-slider"></div>',
+                '<button class="control" aria-disabled="false" aria-label="',
+                    gettext('Volume: Click on this button to mute or unmute this video or press UP or ',
```

Comment:
FYI, our policy in T&L is to always generate the dummy i18n strings on our sandboxes so that we can verify the behavior for each string. If you haven't done that before, you can follow the steps here:

http://edx-developer-guide.readthedocs.org/en/latest/internationalization/i18n.html#coverage-testing

If you revert the splitting of this line, then it may be that you are not introducing new strings and you can skip this step.